feat: add "Examples" navigation link (#26)
electron/electronjs.org-new: scripts/tasks/how-to-examples.md

---
title: 'Examples'

> description: 'Electron examples directly for you to use'

Reviewer: ```suggestion
description: 'A set of examples for common Electron features'
```

slug: examples
hide_title: false
---

# Examples

> In this section we have documented some of the most popular things developers do
> in their Electron applications. The easiest way to run the examples is using
> [Fiddle][fiddle]. To do this, first you will have to download and install the
> right version for your Operating System from [here][fiddle].

Reviewer: I think linking the Fiddle link twice is a bit redundant here

```suggestion
In this section, we have collected a set of guides for common features
that you may want to implement in your Electron application. Each guide
contains a practical example in a minimal, self-contained example app.
The easiest way to run these examples is by downloading [Electron Fiddle][fiddle].
```

> Once you have done this, you can press on the "Open in Fiddle" button that you

Reviewer: ```suggestion
Once Fiddle is installed, you can press on the "Open in Fiddle" button that you
```

will find below code samples like the following one:

```fiddle docs/fiddles/quick-start
window.addEventListener('DOMContentLoaded', () => {
  const replaceText = (selector, text) => {
    const element = document.getElementById(selector)
    if (element) element.innerText = text
  }

  for (const type of ['chrome', 'node', 'electron']) {
    replaceText(`${type}-version`, process.versions[type])
  }
})

> ```

Reviewer: I have an equivalent section meant for the first page you see when you go to `electronjs.org/docs`:

https://github.com/electron/electron/blob/bf04a5f9f83d2af602946e24a2465c724121d77d/docs/tutorial/introduction.md#running-examples-with-electron-fiddle

Author: @erickzhao what content do you think we should have as the landing of examples?

## How to...?

The following are all the "How to?" available at the moment (you can access them
from the sidebar as well). If there is something that you would like to do that

> is not documented, please join our [discord server][] and let us know!

Reviewer: Is there some kind of dynamic loading here since there's no list in the raw markdown?

Author: I was thinking about doing it but then decided it shouldn't have to be part of the MVP (I have a couple other things coming at work :( ) and forgot to update the content.

Author: I've removed the reference. We should definitely do it (I really like the example you provide from Gatsby) but I don't think it's necessary for the MVP. If you are OK with that I'll open an issue to track that effort.

Reviewer: Sounds good to me!

> our [discord server][] and let us know!
> 
> [discord server]: https://discord.com/invite/electron

Reviewer: ```suggestion
our [Discord server][] and let us know!

[Discord server]: https://discord.com/invite/electron
```
